sbin/fsck_ffs/fsck_ffs.8

```diff
@@ -31,7 +33,8 @@
 .Os
 .Sh NAME
 .Nm fsck_ffs ,
-.Nm fsck_ufs
+.Nm fsck_ufs ,
+.Nm fsck_4.2bsd
```

Comment:
This looks to be a separate change...  Can you split it out into its own commit?